### DISCUSSION TITLE
slack invite link invalid

### DISCUSSION THREAD

**Discussion by LambertZhaglog** [harmonycloud/kindling#310] [General]
[slack] This link is no longer active

**Comment by jundizhou** [comment]
update: https://kindling-world.slack.com/join/shared_invite/zt-1fs2yco0i-CMc0yRIqc_jqE~2aHxsNRA#/shared-invite/email
We use wechat more to communicate.If you are in China, you can join wechat group.
![image](https://user-images.githubusercontent.com/92794317/189267494-cc541cbe-5b76-4169-804e-cc478e50bb46.png)

**Reply by llhhbc** [reply]
link is invalid again. @jundizhou

**Reply by dxsup** [reply]
Appreciate your reminder.

Please use this: [Join the Kindling Slack](https://join.slack.com/t/kindling-world/shared_invite/zt-1rmki34l0-JiFpWFR7YhqlXG~mSQQAtg) 

This one will never expire.
